onnxruntime/core/framework/session_options.cc

```diff
@@ -56,6 +56,35 @@ Status SessionOptions::AddExternalInitializers(gsl::span<const std::string> name
   }
   return Status::OK();
 }
+
+Status SessionOptions::AddExternalInitializerFiles(gsl::span<const std::basic_string<ORTCHAR_T>> file_names,
+                                                   gsl::span<void*> array_buffer,
+                                                   gsl::span<const size_t> file_lengths) {
+  const auto init_num = file_names.size();
+  ORT_ENFORCE(init_num == array_buffer.size(), "Expecting same size spans");
+  ORT_ENFORCE(init_num == file_lengths.size(), "Expecting same size spans");
+  external_initializer_files.reserve(external_initializer_files.size() + init_num);
+  std::vector<std::basic_string<ORTCHAR_T>> prefix_list{
```

Comment:
Can this be a static constexpr std::array of std::string_view?

Reply:
Please, at least make it const.

Reply:
are you looking at an old commit?